py/picca/utils.py

```diff
@@ -252,6 +278,8 @@ def desi_convert_transmission_to_delta_files(indir,outdir,lObs_min=3600.,lObs_ma
             deltas[pixnum].append(delta(thid[i],ra[i],dec[i],z[i],thid[i],thid[i],thid[i],cll,civ,None,cfl,1,None,None,None,None,None,None))
         if not nspec is None and sp.sum([ len(deltas[p]) for p in list(deltas.keys())])>=nspec: break
 
+    print('\n')
```

Comment:
this will print two empty lines, is that what you want?

Reply:
I just want to separate the two `sys.stderr.write`. What is the best way?

Reply:
It's ok, eventually we will use one single function for screen output, probably a flushed `print()`